addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -197,11 +197,11 @@ function factory(dependencies) {
          * @private
          * @param {Object} payload
          * @param {mail.thread} payload.channel
-         * @param {integer} payload.invited_by_user_id
+         * @param [integer] payload.invited_by_user_id
```

Comment:
optional should be on the name not on the type

Reply:
Oups